translations/messages+intl-icu.de.yml

```diff
@@ -3083,6 +3090,7 @@ statement.assignment.assigned: "Bearbeitet durch: {name} ({organisation})"
 statement.assignment.override: "Stellungnahme mir zuweisen"
 statement.assignment.release: "Stellungnahme freigeben"
 statement.assignment.unassigned: "Aktuell nicht in Bearbeitung"
+statement.attachments.reference: "Referenz Anhänge"
```

Comment:
What is meant by this? "Referenzanhänge"? "Referenzen auf Anhänge"? Something different?

Reply:
adjusted in [8e46231](https://github.com/demos-europe/demosplan-core/pull/2337/commits/8e4623150eaf7df6c8b8fd08b3cf79a2be0d933d)